test/refract/test-JsonSchema.cc

```diff
@@ -194,6 +227,23 @@ SCENARIO("JSON Schema serialization of NumberElement", "[json-schema]")
             }
         }
     }
+
+    GIVEN("A named NumberElement with content and fixed attribute true")
+    {
+        auto el = from_primitive(42.0);
+        el->element("Baz");
+        el->attributes().set("fixed", from_primitive(true));
```

Comment:
Shouldn’t fixed as string element be placed inside the typeAttributes (array element) inside attributes of the element?

Reply:
Yes, you're right. I completely missed this, thanks :+1: